setuptools/command/sdist.py

```diff
@@ -121,6 +121,14 @@ def __read_template_hack(self):
     if has_leaky_handle:
         read_template = __read_template_hack
 
+    def _add_defaults_optional(self):
+        if six.PY2:
+            sdist_add_defaults._add_defaults_optional(self)
```

Comment:
You can use `super(sdist_add_defaults, self)._add_defaults_optional()`, it works in both Python 2 and 3. It would also be nice to add a comment like "# Python 2 support" or something to that effect so we know to change it when we drop Python 2.

Reply:
Actually I can't because in Python-2 ``sdist_add_defaults`` and ``distutils``' ``sdist`` are old-style classes and using ``super()`` results in ``TypeError: super() argument 1 must be type, not classobj``